src/generate.rs

```diff
@@ -367,6 +377,7 @@ pub(crate) fn get_ram(device: &Device) -> Option<RamRegion> {
             regions.push(RamRegion {
                 range: memory.start as u32..memory.start as u32 + memory.size as u32,
                 is_boot_memory: memory.startup,
+                cores: vec!["".to_owned()], // TODO can we do better?
```

Comment:
Yes we actually need to do better for it to work. It needs to know which core it should use to access this memory. As a default we can use "main". See here: https://github.com/probe-rs/probe-rs/blob/master/probe-rs/targets/nRF52_Series.yaml#L21. The problem is that you use the actual core name, which is not main, so I think we need to change one or the other :) Currently you use the core type as the name which is not a good choice as the multi targets can have multiple cores, even tho you do not implement multi core handling yet.

Reply:
Switched to using "main" as the default core for everything, don't have time to look at multicore atm, sorry!

Reply:
No need to be sorry! I am very thankful you did this already!!